const.py

```diff
@@ -0,0 +1,16 @@
+# Slack custom command secret token
+SLACK_TOKEN = 'LxqD74GgYpSCmQlNsl5MXckS'
```

Comment:
It's better if you read this from env variable as you did for port number. In case you want to use this service in another slack team you won't need to change code.
On the other side, are you planning to support more than one teams? What would you do in that case?